src/asset-management/AaveManager.sol

```diff
@@ -12,8 +12,6 @@ import { IPoolAddressesProvider } from "src/interfaces/aave/IPoolAddressesProvid
 import { IPool } from "src/interfaces/aave/IPool.sol";
 import { IAaveProtocolDataProvider } from "src/interfaces/aave/IAaveProtocolDataProvider.sol";
 
-// TODO: Should probably move the adapter away from AssetManagedPair because
```

Comment:
may I know what did you mean by this? Is it about not using an `AssetManagedPair` interface?

Reply:
idk anymore, can resolve